.clang-tidy

```diff
@@ -48,4 +48,4 @@ WarningsAsErrors:
 -bugprone-unused-raii,
 -clang-diagnostic-deprecated-declarations,
 -clang-analyzer-cplusplus.PlacementNew,
-# TODO Figure out if PlacementNew is a false positive
+'
```

Comment:
This was a bugfix. Seems like clang-tidy doesn't error if it can't parse the file. 👎

Reply:
Sent a PR to prevent this form happening in the future: https://github.com/pre-commit/identify/pull/168